deploy/zookeeper/advanced/zookeeper-create-universal.sh

```diff
@@ -17,5 +17,5 @@ CUR_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
 
 kubectl create namespace ${ZK_NAMESPACE}
 for FILE in ${YAML_FILES_LIST}; do
-    kubectl -n "${ZK_NAMESPACE}" apply -f "${CUR_DIR}/${FILE}"
+    cat "${CUR_DIR}/${FILE}" | kubectl -n "${ZK_NAMESPACE}" apply -f -
```

Comment:
This change is not needed. kubectl apply -f <filename.yaml> is a kubernetes way.

Reply:
if you use this shell script on windows with Cygwin, kubectl will be a native win32 application, but "${CUR_DIR}/${FILE}" will be a `/cygdrive/c/XXXXX/`, so Cygwin `cat.exe` will allow this path, but kubectl.exe not found